hr_attendance_rfid/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_attendance_rfid,access.hr.attendance.rfid,hr_attendance.model_hr_attendance,hr_attendance_rfid.group_hr_attendance_rfid,1,1,1,0
+access_hr_employee_rfid,access.hr.employee.rfid,model_hr_employee,hr_attendance_rfid.group_hr_attendance_rfid,1,0,0,0
```

Comment:
In function attendance_action_change() will state of employee be changed. It seems that the device user just only in group_hr_attendance can not write on model hr.employee.

Reply:
You are never writing to hr.employee. The state is a function field in 8.0 which uses the values from hr.attendance where you have the rights.